src/components/dialogs/parameters/security-analysis/security-analysis-parameters-selector.tsx

```diff
@@ -54,14 +54,21 @@ const SecurityAnalysisParametersSelector: FunctionComponent<{
 
                 {TAB_INFO.map((tab, index) => (
                     <TabPanel key={tab.label} value={tabValue} index={index}>
-                        {tabValue === TAB_VALUES.General && (
-                            <ViolationsHidingParameters params={params} updateParameters={updateParameters} />
-                        )}
-                        {tabValue === TAB_VALUES.LimitReductions && params.limitReductions && (
-                            <Grid sx={{ width: '85%' }}>
-                                <LimitReductionsTableForm limits={params.limitReductions} />
-                            </Grid>
-                        )}
+                        <Box display={tabValue === TAB_VALUES.General ? 'block' : 'none'}>
+                            <ViolationsHidingParameters />
+                        </Box>
+
+                        <Box
+                            sx={{
+                                width: '85%',
+                                display:
+                                    tabValue === TAB_VALUES.LimitReductions && params.limitReductions
+                                        ? 'block'
+                                        : 'none',
+                            }}
+                        >
+                            <LimitReductionsTableForm limits={params.limitReductions} />
+                        </Box>
```

Comment:
why ? what's the link with RHF ?

Reply:
we need to keep the inputs of the form always instantiated 
- state can be lost if we mount / unmount the component ( previously state was managed with a 'useState'
- initialisation of the form crashed the app if i remember correctly.

Reply:
I think I prefer the old solution as well, everything should be stored in RHF here, so it should not cause any problem

Reply:
i did a test, you are right, it is working with the old sollution.
i fixed it